src/outfit.c

```diff
@@ -1413,6 +1517,9 @@ static void outfit_parseSAmmo( Outfit* temp, const xmlNodePtr parent )
             temp->u.amm.spin = atof( buf );
             free(buf);
          }
+         /* Load the collision polygon. */
+         buf = xml_get(node);
+         outfit_loadPLG( temp, buf, 0 );
```

Comment:
Should warn if no polygon is available and give a hint or something on how to generate it.